server/src/main/scala/org/apache/livy/server/LivyServer.scala

```diff
@@ -237,11 +237,21 @@ class LivyServer extends Logging {
         server.context.addFilter(holder, "/*", EnumSet.allOf(classOf[DispatcherType]))
         info(s"SPNEGO auth enabled (principal = $principal)")
 
-     case null =>
+      case null =>
         // Nothing to do.
 
       case other =>
-        throw new IllegalArgumentException(s"Invalid auth type: $other")
+        // go down the standard route of hadoop, unknown is expected to be
```

Comment:
I believe this should be `AltKerberosAuthenticationHandler.TYPE` and you also need to update `pom.xml` to include BDS `hadoop-auth-2.7.x` JAR? I need more context on what we are trying to add here b/c `other` is a wide net to catch here.